src/main/conscript/g8/launchconfig

```diff
@@ -2,7 +2,7 @@
   version: 0.7.2
   org: org.foundweekends.giter8
   name: giter8
-  class: giter8.Giter8
+  class: giter8.Giter8App
```

Comment:
Forgot to revert it.

But it would make sense to call the app class Giter8App, and the library class — Giter8. But it will require having launchconfig with `Giter8App` in main repo, if I understood correctly, how conscript works.

If we stick with old naming (app is `Giter8` and library class is `Giter8Engine`), I will edit lauchconfig.